src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -300,7 +300,7 @@ class WorkspaceInvitePage extends React.Component {
         const policyName = lodashGet(this.props.policy, 'name');
 
         return (
-            <ScreenWrapper shouldEnableMaxHeight>
+            <ScreenWrapper>
```

Comment:
Does not this page actually need this fix?

Reply:
Yes, is using the fix, but the fix is applied by default. Using the fix means unset the maxHeight 

Ignoring the maxHeight (FIX)

https://user-images.githubusercontent.com/1676818/231215441-4a605c14-5445-414d-81f0-0636b6700598.mp4

Using the maxHeight

https://user-images.githubusercontent.com/1676818/231215514-ecb0a38c-0da3-4456-ab3a-fb7d1f2fc821.mp4

Reply:
I'm sorry there appears to be some confusion here. What we want is the behaviour shown in the second video, the "Invite" button should be visible.

Reply:
@s77rt got it, so then yes I need to pass the variable so the page behaves as the second video adding it

Reply:
Updated